README.md

```diff
@@ -166,18 +166,19 @@ This extension contributes the following settings:
 - "Show Report on Execute" will show the HTML test report in after each VectorCAST test run.
 - "Decorate Explorer Tree" will add a VC in the right margin of the file explorer tree for those files that have VectorCAST coverage.
 - "Unit Test Location" controls where the extension stores unit test new unit test artifacts.
-- "Verbose Logging" will add more detailed messages to the VectorCAST Test Explorer message pane
+- "Verbose Logging" will add more detailed messages to the VectorCAST Test Explorer message panel
 
 ## Known Issues
 
-- If a VectorCAST configuration file (CCAST\_.CFG) does not exist in the directory chosen
+- If a VectorCAST configuration file (CCAST_.CFG) does not exist in the directory chosen
```

Comment:
The escape was probably correct; it stops Markdown thinking you're starting an italics block (between two `_`)

Reply:
Ahh ok.  It renders fine either way though ...